wren-ai-service/demo/utils.py

```diff
@@ -25,8 +25,12 @@
 load_dotenv()
 
 
-def add_quotes(sql: str) -> str:
-    return sqlglot.transpile(sql, read="trino", identify=True)[0]
+def add_quotes(sql: str) -> Tuple[str, bool]:
+    try:
+        quoted_sql = sqlglot.transpile(sql, read="trino", identify=True)[0]
+        return quoted_sql, True
+    except Exception:
+        return sql, False
```

Comment:
I found the same function in diff utils module, I think we can consider to remove the duplication.